titus-api/src/main/java/com/netflix/titus/api/model/ApplicationSLA.java

```diff
@@ -30,20 +38,29 @@
 
     private final ResourceDimension resourceDimension;
 
+    private final String schedulerName;
+
     /**
      * Total number of instances required by this application. Titus will keep pre-allocated resources to always
      * fulfill this requirement.
      */
     private int instanceCount;
 
-    public ApplicationSLA(String appName,
-                          Tier tier,
-                          ResourceDimension resourceDimension,
-                          int instanceCount) {
+    @JsonCreator
+    public ApplicationSLA(@JsonProperty("appName") String appName,
```

Comment:
Jackson annotation are in ApplicationSlaMixIn. You should not add them here.

Reply:
Fixed. Thanks